content/en/docs/instrumentation/java/exporters.md

```diff
@@ -45,6 +50,46 @@ dependencies {
 
 {{< /tab >}} {{< /tabpane>}}
 
+Under the hood, there are two protocol options supported, each with different
+"sender" implementations.
+
+- `grpc` - gRPC implementation of OTLP exporters, represented by
+  `OtlpGrpcSpanExporter`, `OtlpGrpcMetricExporter`, `OtlpGrpcLogRecordExporter`.
+- `http/protobuf` - HTTP with protobuf encoded payload implementation of OTLP
+  exporters, represented by `OtlpHttpSpanExporter`, `OtlpHttpMetricExporter`,
+  `OtlpHttpLogRecordExporter`.
+
+A sender is an abstraction which allows different gRPC / HTTP client
+implementations to fulfill the OTLP contract. Regardless of the sender
+implementation, the same exporter classes are used. A sender implementations is
+automatically used when it is detected on the classpath. The sender
```

Comment:
add `classpath` to the ignore list in line 4

We can later consider to have it added to the global list as well.